index.js

```diff
@@ -44,15 +44,15 @@ module.exports = function(options) {
   }))
 
   const _cors = cors(options.cors || {})
-  router.get('/resize/:width/:height?', _cors, async (req, res, next) => {
+  router.get('/resize/:width?/:height?', _cors, async (req, res, next) => {
```

Comment:
How do you provide just a height?

Reply:
I've updated the API so width and height are part of the query. It's strange to have these as part of the route.